sidetree/lib/core/versions/1.0/TransactionProcessor.ts

```diff
@@ -31,12 +33,14 @@ export default class TransactionProcessor implements ITransactionProcessor {
   public constructor (
     private downloadManager: DownloadManager,
     private operationStore: IOperationStore,
+    private didTypeStore : IDidTypeStore,
     private blockchain: IBlockchain,
     private versionMetadataFetcher: IVersionMetadataFetcher) {
   }
 
   public async processTransaction (transaction: TransactionModel): Promise<boolean> {
 
+    console.log('1.1');
```

Comment:
what are these logs for?

Reply:
Removed, 
Everything in the 1.0 directory will be a copy of everything I change in latest directory, will update the PR to reflect that